schedule/yast/minimal+base/minimal+base@yast_x86_64.yaml

```diff
@@ -0,0 +1,36 @@
+---
+name:           minimal+base@yast
+description:    >
+  Select a minimal textmode installation by starting with the default and unselecting all patterns
+  except for "base" and "minimal". Not to be confused with the new system role "minimal" introduced with SLE15.
+vars:
+  DEPENDENCY_RESOLVER_FLAG: 1
+  DESKTOP: textmode
+  PATTERNS: base,enhanced_base
+  YUI_REST_API: 1
+schedule:
+  extension_module_selection:
+    - installation/module_registration/skip_module_registration
+  add_on_product:
```

Comment:
this is repeated, it is already in default.

Reply:
removed